includes/class-wc-ecomm-bridge.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Adds the functionality needed to bridge WooCommerce.
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   1.1.7
+ * @version 1.0.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC EComm Bridge
+ */
+class WC_EComm_Bridge {
+
+	/**
+	 * Class Instance.
+	 *
+	 * @var WC_EComm_Bridge instance
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_action( 'plugins_loaded', array( $this, 'initialize' ), 2 );
+	}
+
+	/**
+	 * Add hooks and filters if WooCommerce is active.
+	 */
+	public function initialize() {
+		if ( ! function_exists( 'WC' ) ) {
+			return;
+		}
+
+		add_filter( 'woocommerce_admin_features_to_enable_disable', array( $this, 'filter_wc_admin_enabled_features' ) );
```

Comment:
I think this needs to be update to `woocommerce_admin_features`, since https://github.com/woocommerce/woocommerce-admin/pull/4523 was the landed PR.